src/ShareRoutes.test.jsx

```diff
@@ -15,10 +15,10 @@ const tests = [
   {s: 'http://github.com/org/repo/blob/branch/a/b/c/file.ifc', out: pathBlobAbc},
   {s: 'https://www.github.com/org/repo/blob/branch/file.ifc', out: pathBlob},
   {s: 'https://github.com/org/repo/blob/branch/file.ifc', out: pathBlob},
-  {s: 'github.com/org/repo/blob/branch/file.ifc', out: pathBlob},
```

Comment:
The use case here is copying text from apps/sites that hide the protocol.  It's a nice-to-have

Reply:
I don't know.  I feel like server will upgrade to ssl if it wants and we don't have any private data per se, so http for me